core/circuitbreaker/rule.go

```diff
@@ -61,7 +61,14 @@ type Rule struct {
 	// that can trigger circuit breaking.
 	MinRequestAmount uint64 `json:"minRequestAmount"`
 	// StatIntervalMs represents statistic time interval of the internal circuit breaker (in ms).
+	// Currently the statistic is collected by sliding window.
```

Comment:
`Currently the statistic interval is collected by sliding window.`

Reply:
Updated.